source/includes/_account.md

```diff
@@ -173,12 +173,14 @@ The body of the request is a JSON string specifying any attributes of the resour
 | **timeZone** | "UTC" |
 | all others | null |
 
-All attributes are optional, so it is possible to quickly create a shell account. This account could be used for test purposes, or so its attributes can be filled in later. This also ensures the initial account contains no PII. Note, however, that the `currency` attribute must have a non-null value, which can be added later, for invoicing to work properly.
+All attributes (other than name) are optional, so it is possible to quickly create a shell account. This account could be used for test purposes, so its attributes can be filled in later. This also ensures the initial account contains no PII. Note, however, that the `currency` attribute must have a non-null value. This cannot be added later and is required for invoicing to work properly.
 
 A few fields are not updatable; they can only be set once when creating the original `Account`. These include  `externalKey`, `currency`, `timeZone`, and `referenceTime`. In addition the `billCycleDayLocal` can be updated but only **once**, that is one can create an `Account` without specifying the `billCycleDayLocal` and later update its value; this, in particular allows the system to update its value to a good default, that is one that will avoid leading prorations, when creating the first subscription.
 
 The `accountId` and audit logs are generated by the system and cannot be set or modified by the user.
 
+In order to create an account as a child account, the `parentAccountId` attribute needs to be specified with the `id` of the parent account. Also, the `isPaymentDelegatedToParent` attribute can be specified with a true/false value. A `true` indicates that the child account will be responsible for paying its own invoices while a `false` indicates that the parent account will be responsible for paying the child account invoices. 
```

Comment:
Other way around: `true` -> parent pays for the invoices.

Reply:
Fixed